schemas/iso19139/src/main/plugin/iso19139/process/onlinesrc-add.xsl

```diff
@@ -25,11 +25,19 @@
 <!--
 Processing to insert or update an online resource element.
 Insert is made in first transferOptions found.
+
+Note: It assumes that it will be adding new items in
+      the first /gmd:distributionInfo
+      and first /gmd:MD_Distribution
+      and first /gmd:transferOptions
 -->
 <xsl:stylesheet xmlns:gmd="http://www.isotc211.org/2005/gmd"
                 xmlns:gco="http://www.isotc211.org/2005/gco"
                 xmlns:gmx="http://www.isotc211.org/2005/gmx"
                 xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                xmlns:digestUtils="java:org.apache.commons.codec.digest.DigestUtils"
+                xmlns:exslt="http://exslt.org/common"
```

Comment:
```suggestion
                xmlns:exslt="http://exslt.org/common"
                exclude-result-prefixes="#all"
```

Otherwise the namespaces end in the `gmd:distributionInfo` element:

```
<gmd:distributionInfo xmlns:xs="http://www.w3.org/2001/XMLSchema"
                         xmlns:digestUtils="java:org.apache.commons.codec.digest.DigestUtils"
                         xmlns:exslt="http://exslt.org/common">
...
```

Reply:
Test good. See my other comment for the result